zappa_settings.json

```diff
@@ -0,0 +1,14 @@
+{
+    "dev": {
+        "django_settings": "CoskunLab.settings",
+        "profile_name": null,
+        "project_name": "coskunlabdjango",
+        "runtime": "python3.8",
+        "s3_bucket": "zappa-vc1f6vxja",
+        "aws_region": "us-west-1",
+        "environment_variables": {
+        	"Access key ID": "AKIAZQSBOKUYWFJXU5ZY",
```

Comment:
We might have to change this so this information is;t just available online...

Reply:
maybe create a temporary secret keys file that stores the keys and add it to git ignore. Then import the variables from here until we come up with a better option. Otherwise everything looks good